Makefile

```
@@ -89,9 +93,12 @@ _print_vars:
 _create_namespace:
 	$(K8S_CLI) create namespace $(NAMESPACE) || true
 
-_generate_related_images_env:
-	@mkdir -p $(INTERNAL_TMP_DIR)
+_gen_configuration_env:
+	mkdir -p $(INTERNAL_TMP_DIR)
 	echo "export RELATED_IMAGE_devworkspace_webhook_server=$(IMG)" > $(RELATED_IMAGES_FILE)
+ifeq ($(PLATFORM),kubernetes)
+	echo "export WEBHOOK_SECRET_NAME=devworkspace-operator-webhook-cert" > $(RELATED_IMAGES_FILE)
```

```suggestion
	echo "export WEBHOOK_SECRET_NAME=devworkspace-operator-webhook-cert" >> $(RELATED_IMAGES_FILE)
```

You killed `RELATED_IMAGE_devworkspace_webhook_server` :skull:

---

Good catch, that's a woopsie :smile: